egs/vaevc/template/run.sh

```diff
@@ -24,7 +24,7 @@ n_jobs=10 # number of parallel jobs
 n_gpus=1  # number of gpus
 
 # directory settings
-wavdir=downloads/wav # directory to save downloaded wav files
+downloaddir=        # directory to save downloaded wav files
```

Comment:
When $downloaddir is not defined, $wavdir is placed in ```/wav``` not ```./wav``` isn't it?

Reply:
I see. I will specify a default value for `downloaddir`.